### PR TITLE
Update cross_building_with_conan.rst

### PR DESCRIPTION
**Pull request by noksel**
fix missing --output-folder=build. Further in the text there is "cd build"

### CODE REVIEW

File: tutorial/consuming_packages/cross_building_with_conan.rst [hunk 1]
```diff
@@ -168,7 +168,7 @@ dependency built for *armv7hf* architecture and a **cmake/3.22.6** version that
 
 .. code-block:: bash
     
-    $ conan install . --build missing -pr:b=default -pr:h=./profiles/raspberry
+    $ conan install . --output-folder=build --build missing -pr:b=default -pr:h=./profiles/raspberry
```
Comment (memsharded) [inline tutorial/consuming_packages/cross_building_with_conan.rst:171]: Not necessary, the recipe contains a ``cmake_layout(self)`` that automatically defines the "build" folder

**Review by memsharded — CHANGES_REQUESTED** [review]
Hi @noksel 
thanks for your contribution.

However, I'd say this change is not necessary, could you please double check? If the recipe contains a ``layout()`` then defining the ``--output-folder`` is typically not necessary

File: tutorial/consuming_packages/cross_building_with_conan.rst (hunk 1 above)
Comment (noksel) [reply tutorial/consuming_packages/cross_building_with_conan.rst:171]: oh. Sorry. I have used conanfile.txt instead conanfile.py

**Comment by noksel** [comment]
oh. Sorry. I have used conanfile.txt instead conanfile.py

**Pull request closed without merging**